README.md

```diff
@@ -2,14 +2,26 @@ _NOTE: The interfaces of flux-accounting are being actively developed and are no
 
 ## flux-accounting
 
-Development for a bank/accounting interface for the Flux resource manager. Writes and saves job step history, user account information, and priority calculation to persistent storage using Python's SQLite3 API. 
+Development for a bank/accounting interface for the Flux resource manager. Writes and saves user account information to persistent storage using Python's SQLite3 API.
 
-##### job step history
+##### Install instructions
 
-Running as a cron job, `write_jobs.py` uses a flux-core RPC to retrieve job data that have recently transitioned to an INACTIVE state in the past _x_ minutes. It parses the job record data and writes it to a SQLite database, which can be queried at a later time. 
+This repository uses `pandas` for data display information. To install this dependency, along with any other packages specified in **requirements.txt**, and also run unit tests, you can run `make check`:
 
-##### user account information
+```
```

Comment:
Are there any other dependencies? Does yours depend on flux-core or flux-sched at this point? I think I saw one of the module depends on flux-core's python binding. If so, do you want to say "flux-sched requires an installed flux-core package"?

Reply:
Thanks for catching that. At this point, flux-accounting doesn't depend on flux-core's python binding. I was going to decorate the main function with flux-core's `util.CLIMain` decorator, but I decided to just include my own logger for now, which uses Python's standard `logging` module.

Reply:
Ah. Cool. This can make the testing of your core logic easier!